org.yuzu_emu.yuzu.json

```diff
@@ -14,7 +14,9 @@
         "--share=ipc",
         "--filesystem=xdg-run/app/com.discordapp.Discord:create",
         "--filesystem=home:ro",
-        "--filesystem=/run/media:ro"
+        "--filesystem=/run/media:ro",
+        "--talk-name=org.freedesktop.login1.Manager",
+        "--talk-name=org.freedesktop.portal.Inhibit"
```

Comment:
I think those are already included in the default Flatpak scope?

```
[📦 org.yuzu_emu.yuzu org.yuzu_emu.yuzu]$ qdbus org.freedesktop.portal.Desktop /org/freedesktop/portal/desktop
... snip ...
property read uint org.freedesktop.portal.Inhibit.version
signal void org.freedesktop.portal.Inhibit.StateChanged(QDBusObjectPath session_handle, QVariantMap state)
method QDBusObjectPath org.freedesktop.portal.Inhibit.CreateMonitor(QString window, QVariantMap options)
method QDBusObjectPath org.freedesktop.portal.Inhibit.Inhibit(QString window, uint flags, QVariantMap options)
method void org.freedesktop.portal.Inhibit.QueryEndResponse(QDBusObjectPath session_handle)
... snip ...
```

Reply:
i don't know what this output means but if it implies i should remove the arg i can do that.

Reply:
> i don't know what this output means but if it implies i should remove the arg i can do that.

The output is the introspection of the `org.freedesktop.portal.Desktop` endpoint in the Flatpak sandbox. If the endpoint is unavailable in the sandbox, it will not appear in this output.

`org.freedesktop.login1.Manager` is indeed not available since it's a system endpoint. Usually, this is not needed in the Flatpak context; however, I heard this might be needed on GNOME + Wayland (GNOME + X11 does not need this). If you are using this combination, you can help me verify whether this is the case.